spacy/util.py

```diff
@@ -93,6 +93,7 @@ class registry(thinc.registry):
     # environment. spaCy models packaged with `spacy package` will "advertise"
     # themselves via entry points.
     models = catalogue.create("spacy", "models", entry_points=True)
+    annotation_setters = catalogue.create("spacy", "annotation_setters", entry_points=True)
```

Comment:
Hm, now I started wondering whether this is the right move, as the signature of spaCy's `Tok2Vec` `annotation_setter` is different to the one used in `spacy-transformers`. But it would also be weird to make two separate registries for them, no?